internal/utils/utils.go

```diff
@@ -263,15 +265,15 @@ func WriteToWorkingDirFile(toFile string, value []byte) error {
 }
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rgen = rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
we actively do not want this: "this source is not safe for concurrent use"
https://pkg.go.dev/math/rand#NewSource

Reply:
Moved into test/helpers given we no longer use this (was used for Logstore) but I do not think it matters.  We are not relying on any consistently generated "Random" name